### DISCUSSION TITLE
Join as a student

### DISCUSSION THREAD

**Discussion by hillmanlam** [community/community#62238] [GitHub Education]
### Hi everyone,

Hello All, I attempted to join as a student but unable to complete the application because 1. Signup detects that I'm a student outside the university's location (It's because I'm an online student) and 2. Upload documents only allow by taking a picture of student ID (Don't have because I'm an online student). Look forward to any suggestions. Thank you!

**Comment by nirav-gajera** [comment]
**Try using a fee receipt or a screenshot of your college portal. Sometimes, this works. Make sure that your username and other details match the ones you provided to the college.**
